src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.NetSecurityNative.cs

```diff
@@ -0,0 +1,126 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class NetSecurityNative
+    {
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurityNative_ReleaseGssBuffer")]
+        internal static extern void ReleaseGssBuffer(
+            IntPtr bufferPtr,
+            UInt64 length);
+
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurityNative_DisplayStatus")]
+        internal static extern unsafe Status DisplayStatus(
+            out Status minorStatus,
+            Status statusValue,
+            bool isGssMechCode,
+            GssBuffer* buffer);
```

Comment:
As noted earlier, rather than using pointers for things like this, if you want to you should be able to use ```ref``` or ```out```... it'll still be marshaled as a pointer so the native code wouldn't change.